docs/core/install/includes/linux-libgdiplus-general.md

```diff
@@ -0,0 +1,4 @@
+
+If the .NET app uses the *System.Drawing.Common* assembly, libgdiplus will also need to be installed.
+Because [*System.Drawing.Common* is no longer supported on Linux](../../compatibility/core-libraries/6.0/system-drawing-common-windows-only.md),
+this only works on .NET 6 and requires setting the `System.Drawing.EnableUnixSupport` runtime configuration switch.
```

Comment:
Regarding this include, we don't usually insert new lines within the same paragraph, just let it run and naturally wrap.

Shouldn't this be more in your face though? I believe with .NET 7 libgdiplus is no longer supported, correct? And the recommendation is to move to another library such as SkiaSharp?

I also added a header to the file.

```suggestion
---
author: adegeo
ms.author: adegeo
ms.date: 12/27/2022
ms.topic: include
---

Starting with .NET 6, the *System.Drawing.Common* library is no longer supported on Linux. If your app has a dependency on this library, you can install *libgdiplus*, but you'll need to set the `System.Drawing.EnableUnixSupport` runtime configuration switch to force the app to use it.

> ![WARNING]
> Starting with .NET 7, libgdiplus is no longer supported by .NET and the app must migrate to an alternative drawing library.

For more information about *System.Drawing.Common* and *libgdiplus*, see [System.Drawing.Common only supported on Windows](../../compatibility/core-libraries/6.0/system-drawing-common-windows-only.md).
```

Reply:
> Shouldn't this be more in your face though? I believe with .NET 7 libgdiplus is no longer supported, correct? And the recommendation is to move to another library such as SkiaSharp?

These articles are about dependencies. I think that most people wanting to learn about those don't need to know about the state of `System.Drawing.Common`/`ligdiplus`. And those who need to know almost certainly already do, since using `System.Drawing.Common` on Unix warns and requires a special flag on .Net 6 and doesn't work at all on .Net 7.

So my goal was to downplay any information about `libgdiplus`, especially since trying to install it when not needed seems to be an issue for some (like in the linked https://github.com/dotnet/docs/issues/33101).